.github/workflows/test.yml

```diff
@@ -50,10 +51,15 @@ jobs:
 
       - run: just lint-device --release --locked
       - run: just build-device ${{ matrix.board }} --locked
+      - run: just save-image ${{ matrix.board }}
       - uses: "./.github/actions/upload-artifact"
         with:
           name: frostsnap-${{ github.sha }}.elf
           path: target/riscv32imc-unknown-none-elf/release/v2
```

Comment:
:question: can we remove this artifact. I guess the fewer artifacts the better.

Reply:
Removed!